source/system/threading/ThreadId.ooc

```diff
@@ -0,0 +1,26 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+import native/[ThreadUnix, ThreadWin32]
+include pthread | (_POSIX_C_SOURCE=200809L)
+
+version (unix || apple) {
+	ThreadId: cover from PThread {
```

Comment:
You could also add this directly to Thead.ooc, as it is not very useful without `Thread`